src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -113,7 +114,7 @@ def get_metadata(self):
 
     def run_conversion(
         self,
-        nwbfile_path: OptionalFilePathType = None,
+        nwbfile_path: Optional[FilePathType] = None,
```

Comment:
I think this is a good change. I feel that `OptionalFilePathType` is unecessary indirection and using `Optional[FilePathType]` is equally clear. We could go all the way to use `Optional[Union[Path, str]]` but maybe that's too verbose? This seems like a good compromise.

We have this `OptionalFilePathType` in some other places as well though, maybe it would be good to change it everywhere?

Reply:
I prefer to keep the distinction between FilePathType and FolderPathType so that we can offer different validation and user interfacing tools depending on if the user needs to select a file or a folder

Reply:
Yes, I'd like to phase out all built-in Optional types and simply use Optional[] throughtout

Reply:
This makes sense to me, FilePathType and FolderPathType is a meaningful distinction that we want to hint for the readers (e.g. pass the path of a folder/directory vs pass the path of a file).